extensions/amp-video/0.1/amp-video.js

```diff
@@ -14,26 +14,35 @@
   * limitations under the License.
   */
 
+import {ampdocServiceFor} from '../../../src/ampdoc';
```

Comment:
ads mentioned, let's do a `git mv` for `amp-video.js` , `test-amp-video` first to preserve the history and then make the changes.